rust-wasm-id-allocator/distributed-id-allocator/src/compressor.rs

```diff
@@ -142,6 +167,9 @@ impl IdCompressor {
         }
     }
 
+    /// Finalizes the supplied range of IDs (which may be from either a remote or local session).
+    /// This method encapsulates the total order broadcast logic which guarantees state synchronization between multiple networked compressors.
```

Comment:
I'd say that, rather than encapsulating TOB, this method relies on it. It must only be called with ranges that have been ordered by a service and that should be noted explicitly (probably supplants the last sentence).